site/source/user-manual/backups/backup-setup/backup-synology.rst

```diff
@@ -13,19 +13,19 @@ Synology Network Folder
 Setup Network Folder
 --------------------
 
-#. In Synology UI, go to *Control Panel > Shared Folder* and select the folder you want to use.
+#. In the Synology UI, go to *Control Panel > Shared Folder* and choose the folder you want to use as the destination for the backup.
 
-    .. note:: Do not select "Use this shared folder" as folder encryption is not supported at this time
+    .. note:: Do not select an encrypted folder. Encrypted folders on Synology enforce a character limit of 143 characters.  The StartOS backup uses folder/file names that are longer than 143 characters.  The backup process will fail if you try to backup to an encrypted folder.
         
-#. Back in Synology UI, go to *Control Panel > File Services > SMB* and click the SMB drop down button if it isn't already selected. Ensure that "Enable SMB service" is checked.
+#. Still in the Synology UI, go to *Control Panel > File Services > SMB* and click the SMB tab if it isn't already selected. Ensure that "Enable SMB service" is checked.
 
-#. Under Advanced Settings on the same dropdown, set "Min SMB protocol" to SMB2 and "Max SMB protocol" to SMB3
+#. Under Advanced Settings on the same tab, set "Min SMB protocol" to SMB2 and "Max SMB protocol" to SMB3
 
-#. On the rsync tab, under File Services -> turn on rsync protocol.
+#. Also on the SMB tab, take note of your device name.  Just under "Note" in a pale blue box, you will see "PC (Windows Explorer): " and "Mac (Finder):". These both provide network addresses that contain your device's name. This device name is the "Hostname" you will need to provide within the StartOS "New Network Folder" dialog in step 3 of the "Connect StartOS" section below.
```

Comment:
Is something shown for Linux?

Reply:
Nope.  Looks like this
<img width="815" alt="image" src="https://github.com/Start9Labs/documentation/assets/733873/48b03299-acca-469c-b6a7-1b6ae692080c">